README.md

```diff
@@ -43,3 +43,35 @@ let expected = vec![
 
 assert_eq!(resampled, expected);
 ```
+
+
+## Usage in Python
+
+To resample a stream of samples to a given interval, you can use the `Resampler`
+class.
+The construction of a resampler expects an interval (`datetime.timedelta`),
+a `ResamplingFunction`, a `max_age_in_intervals` parameter to control the
+maximum age of a sample (skips all samples if set to `0`), and a `start` parameter to set the start time of the
+first resampled sample.
+
+```python
+import datetime as dt
+from frequenz_resampling import Resampler, ResamplingFunction
```

Comment:
I guess there is no way to export the package in python as `frequenz.resampling` instead? If it was possible that would be ideal, so we have a consistent `import`s scheme.

Reply:
yep, that works :)

Reply:
Aweome!